getpfp.py

```diff
@@ -0,0 +1,75 @@
+# -*- coding: future_fstrings -*-
+
+#    Friendly Telegram (telegram userbot)
+#    Copyright (C) 2018-2019 The Authors
+
+#    This program is free software: you can redistribute it and/or modify
+#    it under the terms of the GNU Affero General Public License as published by
+#    the Free Software Foundation, either version 3 of the License, or
+#    (at your option) any later version.
+
+#    This program is distributed in the hope that it will be useful,
+#    but WITHOUT ANY WARRANTY; without even the implied warranty of
+#    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#    GNU Affero General Public License for more details.
+
+#    You should have received a copy of the GNU Affero General Public License
+#    along with this program.  If not, see <https://www.gnu.org/licenses/>.
+
+import logging
+
+from .. import loader, utils
+
+logger = logging.getLogger(__name__)
+
+
+def register(cb):
+    cb(GetPPMod())
+
+
+class GetPPMod(loader.Module):
+    """Gets the profile photos of the target"""
+    def __init__(self):
+        self.name = _("Profile Photo Fetcher")
+
+    async def client_ready(self, client, db):
+        self.client = client
+        self.db = db
+
+    async def getpfpcmd(self, message):
+        """Fetches the profile photos of replied users, channels or chats"""
+        id = utils.get_args_raw(message)
+        user = await message.get_reply_message()
+        chat = message.input_chat
```

Comment:
You don't use this variable properly. It should be `(await message.get_reply_message()).input_sender if message.is_reply else message.input_chat`, then you can remove all the later conditionals